resteasy-core-spi/src/main/java/org/jboss/resteasy/resteasy_jaxrs/i18n/LogMessages.java

```diff
@@ -325,6 +325,14 @@ public interface LogMessages extends BasicLogger
    @Message(id = BASE + 370, value = "Failed to load: %s")
    void failedToLoad(@Cause Throwable cause, String resourceName);
 
+   @LogMessage(level = Level.ERROR)
+   @Message(id = BASE + 375, value = "Handling an uncaught exception with the default exception mapper.")
+   void defaultExceptionMapper(@Cause Throwable cause);
+
+   @LogMessage(level = Level.ERROR)
+   @Message("Handling an uncaught exception with the default exception mapper on resource %s.")
```

Comment:
You don't want a message id here?

Reply:
It's inherited from the `void defaultExceptionMapper(@Cause Throwable cause);`